source/include/azure_iot.h

```diff
@@ -0,0 +1,153 @@
+/* Copyright (c) Microsoft Corporation. All rights reserved. */
+/* SPDX-License-Identifier: MIT */
+
+/**
+ * @file azure_iot.h
+ *
+ * @brief Azure IoT FreeRTOS middleware common APIs and structs
+ *
+ * @note You MUST NOT use any symbols (macros, functions, structures, enums, etc.)
+ * prefixed with an underscore ('_') directly in your application code. These symbols
+ * are part of Azure SDK's internal implementation; we do not document these symbols
+ * and they are subject to change in future versions of the SDK which would break your code.
+ *
+ */
+
+#ifndef AZURE_IOT_H
+#define AZURE_IOT_H
+
+/* AZURE_IOT_DO_NOT_USE_CUSTOM_CONFIG allows building the azure iot library
+ * without a custom config. If a custom config is provided, the
+ * AZURE_IOT_DO_NOT_USE_CUSTOM_CONFIG macro should not be defined. */
+#ifndef AZURE_IOT_DO_NOT_USE_CUSTOM_CONFIG
+    /* Include custom config file before other headers. */
+    #include "azure_iot_config.h"
+#endif
+
+/* Include config defaults header to get default values of configs not
+ * defined in azure_iot_mqtt_config.h file. */
+#include "azure_iot_config_defaults.h"
+
```

Comment:
why are there blank lines between the includes?

Reply:
separate logical includes from different places